changelog/unreleased/7862

```diff
@@ -0,0 +1,5 @@
+Change: The client uploads chunks even so the server repports lack of support
```

Comment:
Do you mean `even though`? 

Also I honestly can't tell from this change what was the "previous" handling was and what happens "after" this change.

Reply:
thx